source/System.cpp

```diff
@@ -704,6 +705,16 @@ void System::LoadObject(const DataNode &node, Set<Planet> &planets, int parent)
 			object.offset = child.Value(1);
 		else if(child.Token(0) == "object")
 			LoadObject(child, planets, index);
+		else if(child.Token(0) == "defense" && child.Size() >= 2)
+		{
+			//Stations don't need anti missile because they can't be attacked.
+			if(!GameData::Outfits().Get(child.Token(1))->AntiMissile())
+			{
+				object.launcher = GameData::Outfits().Get(child.Token(1));
+				if(child.Size() > 2)
+					object.reload = 1.0/atof(child.Token(2).c_str());
```

Comment:
`1. / child.Value(2)`, and you should check that it is not `0` (logging an error if it is)